### DISCUSSION TITLE
ErrorFeedbackCard

### DISCUSSION THREAD

**Discussion by MontseTelefonica** [Telefonica/mistica-design#1449] [Proposals]
### Goal

Para poder crear la experiencia deseada en Novo app, necesitamos 2 UI distintos para los estados de empty state y error.

Vivo quiere que en caso de empty state se use la card con icono de color brand y en caso de error se use una card con icono de color error.

<img width="656" alt="MicrosoftTeams-image (318)" src="https://github.com/Telefonica/mistica-design/assets/95479574/af8e41fa-6293-45ef-850c-328004399e4e">
<img width="402" alt="MicrosoftTeams-image (317)" src="https://github.com/Telefonica/mistica-design/assets/95479574/d860dd41-9d8f-49c1-a760-9b7f2b7641b8">

### Context

Para poder tener 2 UI distintos (icono y color diferente) para las cards de empty state y las cards de error de módulo que plantea Vivo en Novo App, necesitaríamos disponer de una ErrorFeedbackCard que siguiese el mismo patrón de color de icono que la errorFeedbackScreen.

<img width="407" alt="MicrosoftTeams-image (316)" src="https://github.com/Telefonica/mistica-design/assets/95479574/5c770466-ea9f-421f-9ae8-5233b27a762c">

Ya que actualmente en CPS se usa la misma card para ambos casos (empty state card) no podemos realizar la diferenciación de color.

### Impact

Disponer de esta card generaría un cambio global, ya que al usarla en Vivo para los casos de error, habría que usarla para los errores del resto de las Obs. 

### Discussion Questions

-

### Do you have a visual proposal?

https://www.figma.com/file/eE87dEhhkHw8esqhfvoZ0c/Figma-for-Mistica-Discussions?type=design&node-id=0-1&mode=design

**Comment by MontseTelefonica** [comment]
Esta necesidad de Vivo ya sido cubierta con un IF en desarrollo por lo que esto ya no es urgente, aunque estaría genial tenerlo en cuenta :)

**Reply by HectorSancho** [reply]
Dejo un link a las pruebas exploratorias y a las opciones a las que hemos llegado después de la reunión :)

https://www.figma.com/file/eGjt8omC871M7SbFcBFQnC/Untitled?type=design&node-id=0%3A1&mode=design&t=TcmfBwPrOzQDMMmd-1

**Comment by aweell** [comment]
Añado aquí las últimas actualizaciones sobre el tema feedbacks

* Por un lado se mantiene el uso del feedback error tal y como estaba hasta ahora
* Para aquellos casos en los que el error no se muestra a pantalla completa, se creará el componente feedbackErrorCard

Aquí las specs en desarrollo del componente en cuestión: 

https://www.figma.com/file/LpjgnU1xpmEzQvlXKwAHmn/branch/BkmDYewmJAwzEt0vSNPBwm/%F0%9F%94%B8-Feedbacks-Specs?type=design&node-id=2415%3A7999&mode=design&t=drWFTAYzmqTCJr5f-1

Este componente está pensado para funcionar sobre cualquier tipo de background,

**Reply by yceballost** [reply]
@aweell este discussion quedaría cerrada?
